cmake/winml_unittests.cmake

```diff
@@ -80,11 +80,27 @@ function(get_winml_test_api_src
   output_winml_test_api_src
 )
   file(GLOB winml_test_api_src CONFIGURE_DEPENDS
-      "${winml_test_src_path}/api/*.h"
-      "${winml_test_src_path}/api/*.cpp"
+      "${winml_test_src_path}/api/APITest.h"
+      "${winml_test_src_path}/api/LearningModelAPITest.h"
+      "${winml_test_src_path}/api/LearningModelBindingAPITest.h"
+      "${winml_test_src_path}/api/LearningModelSessionAPITest.h"
+      "${winml_test_src_path}/api/LearningModelAPITest.cpp"
+      "${winml_test_src_path}/api/LearningModelBindingAPITest.cpp"
+      "${winml_test_src_path}/api/LearningModelSessionAPITest.cpp")
+  
+  if (NOT winml_is_inbox)
+      file(GLOB winml_redist_only_api_src CONFIGURE_DEPENDS
+      "${winml_test_src_path}/api/RawApiHelpers.h"
```

Comment:
This would make the redist build to only test the Raw Api Tests. Is it enough that just the inbox build runs the model/binding/session api tests or should the redist build also test those as well?

Reply:
Hmm, the redist will build the raw api tests in addition to the other tests?